### PR TITLE
DBZ-2374 Fix featured blog post script & CI exit code propagation.

### PR DESCRIPTION
**Pull request by Naros**
https://issues.redhat.com/browse/DBZ-2374

### CODE REVIEW

File: _ext/featured_posts.rb [hunk 1]
```diff
@@ -35,7 +35,7 @@ def execute(site)
         # Guarantee that each page has a date value for sorting purposes
         featured.each do |p|
           if p.date.nil?
-            if ( p.relative_source_path =~ /^#{@path_prefix}\/(20[01][0-9])-([01][0-9])-([0123][0-9])-([^.]+)\..*$/ )
+            if ( p.relative_source_path =~ /^#{@path_prefix}\/(20[0123456789][0-9])-([01][0-9])-([0123][0-9])-([^.]+)\..*$/ )
```
Comment (jpechane) [inline _ext/featured_posts.rb:38]: Would not be easier to change it to `[0-9]`?

Comment (gunnarmorling) [inline _ext/featured_posts.rb:38]:
```suggestion
            if ( p.relative_source_path =~ /^#{@path_prefix}\/(20[0-9][0-9])-([01][0-9])-([0123][0-9])-([^.]+)\..*$/ )
```

Comment (gunnarmorling) [reply _ext/featured_posts.rb]: Good catch; but please just do these kinds of changes yourself, no need for another round of forth and back :) I just changed it and will merge it.

**Comment by gunnarmorling** [comment]
Thanks, @Naros. Weird, we had the same issue elsewhere before; someone was just not ready for the 20ies, it seems :)

**Comment by Naros** [comment]
Thanks for tweaking the regex & merging.

**Pull request merged**